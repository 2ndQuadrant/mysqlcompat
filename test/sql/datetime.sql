\set ECHO none
BEGIN;
\i sql_bits/datetime.sql
\set ECHO all

SELECT to_days('0001-01-01');
SELECT from_days(1);
SELECT to_days('2011-08-15');
SELECT from_days(734729);
SELECT from_days(to_days(now()::date)) = now()::date;

ROLLBACK;
