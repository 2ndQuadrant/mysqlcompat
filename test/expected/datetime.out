\set ECHO 0
SELECT to_days('0001-01-01');
 to_days 
---------
     366
(1 row)

SELECT from_days(1);
   from_days   
---------------
 01-02-0001 BC
(1 row)

SELECT to_days('2011-08-15');
 to_days 
---------
  734729
(1 row)

SELECT from_days(734729);
 from_days  
------------
 08-15-2011
(1 row)

SELECT from_days(to_days(now()::date)) = now()::date;
 ?column? 
----------
 t
(1 row)

ROLLBACK;
